File: matchmaker/server/src/main/java/ru/ifmo/se/dating/matchmaker/api/HttpSuggestionsApi.java
package ru.ifmo.se.dating.matchmaker.api;

import java.util.Set;

import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Service;

import ru.ifmo.se.dating.matchmaker.api.generated.SuggestionsApiDelegate;

@Service
class HttpSuggestionsApi implements SuggestionsApiDelegate {

    @Override
    public ResponseEntity<Set<Long>> suggestionsGet(Long limit) {
        return ResponseEntityStub.create();
    }
}

File: matchmaker/server/src/main/kotlin/ru/ifmo/se/dating/matchmaker/api/HttpSuggestionsApi.kt
package ru.ifmo.se.dating.matchmaker.api

import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Service
import ru.ifmo.se.dating.matchmaker.api.generated.SuggestionsApiDelegate

@Service
internal class HttpSuggestionsApi : SuggestionsApiDelegate {
    override fun suggestionsGet(limit: Long): ResponseEntity<Set<Long>> =
        ResponseEntityStub.create()
}
